test/cmd/create.sh

```diff
@@ -165,7 +165,7 @@ run_kubectl_create_validate_tests() {
   kube::log::status "Testing kubectl create --validate=true"
   # create and verify
   output_message=$(! kubectl create -f hack/testdata/invalid-deployment-unknown-and-duplicate-fields.yaml --validate=true 2>&1)
-  kube::test::if_has_string "${output_message}" 'error validating data'
```

Comment:
move this to one of the testing commits

Reply:
flipping the feature gat to beta will break this test unless it is also updated. 

Are you suggesting we split this change into two commits, one where we delete the test (in the commit that updates tests to work for both enabled/disabled), and a second commit where we re-add the test with the new otuput message (the last where we add tests specifically for beta)? Or are you suggesting we do it another way?

Reply:
ok, I've modified it so it passes for either enabled or disabled and moved it to the testing commit. lmk if this is what you were expecting

Reply:
yeah, something that passes with either message